lib/helpstation/fetchers.rb

```diff
@@ -2,6 +2,50 @@ module Helpstation
   module Fetchers
     NotFoundError = Class.new(StandardError)
 
+    # Helps to run fetchers in parallel
+    #
+    # All output from the fetchers are merged and then returned. This means
+    # that if two fetchers return results with the same key then one will
+    # overwrite the other.
+    #
+    # @example
+    #   process ParallelFetcher[
+    #     OperatorFetcher,
+    #     VisitorFetcher
+    #   ], NOT_FOUND_ERROR
+    #
+    class ParallelFetcher < Processor
```

Comment:
Yes, that's also one option. I chose fetcher at the moment because in fetchers we only add to the input and never remove anything from there. I guess I can make a note about it so it can be used with other processors also.